bugs/gh-575.txt

```diff
@@ -0,0 +1,246 @@
+https://github.com/canonical/cloud-init/commit/f680114446a5a20ce88f3d10d966811a774c8e8f
+
+=== Begin SRU Template ===
+[Impact]
+New functionality was added  allowing a root user to validate the userdata
+provided to the instance by running a new CLI option.
+
+[Test Case]
+
+There is an upstream test for this behavior:
+
```

Comment:
I think the test link is missed

Reply:
That's what I get for a last minute push at the end of the workday :wink: